questions/runonce.md

```diff
@@ -13,9 +16,18 @@ For this method our first command checks two things: Your condition for running
     execute as @a[scores={matched=0},x=73,y=10,z=3,distance=..1] run say I just entered the area!
     execute as @a store success score @s matched if entity @s[x=73,y=10,z=3,distance=..1]
 
-**Note:** The order in which the commands are executed is important here. In the first command you check your condition and execute the command, and the second command store the success of executing your condition.
+> [!NOTE]
+> The order in which the commands are executed is important here. In the first command you check your condition and execute the command, and the second command store the success of executing your condition.
+
+In order for it to work in Bedrock Edition too, the commands should be:
 
-Besides the player, this could be some kind of global event, for example, it started to rain (1.20.5+):
+    scoreboard players add @a matched 0
+    # if possible the command above run only on first join
+    execute as @a[scores={matched=0},x=73,y=10,z=3,distance=..1] run say I just entered the area!
+    scoreboard players set @a[scores={matched=0},x=73,y=10,z=3,distance=..1] matched 1
+    execute as @a unless entity @s[scores={matched=1},x=73,y=10,z=3,distance=..1] run scoreboard players set @s matched 0
```

Comment:
I think the first command (`scoreboard players add @a matched 0`) is not needed here. `matched 0` will be set after the first run of command blocks by the last command.
Also, it seems that Bedrock uses `r=` instead of `distance=`.
And also I'm not sure how appropriate it is to give Bedrock code between Java examples. Maybe a separate section for Bedrock should be made?

Reply:
the r= vs distance= is explained at the very top of this page. I think this section is giving Java examples, it's just trying to point out that for it to work in bedrock you need to use a different setup (which works in both editions). 

So it's fine as is, maybe we just need to take a look at the explanation part.

We could change this

> In order for it to work in Bedrock Edition too, the commands should be:

into this

> The previous example doesn't work in Bedrock Edition. Here is a setup that works in both Java and Bedrock Edition:

Reply:
So I remove the first command right?
And about the second part it already says it in the note at the start of the file, but I changed it so it is more easy to understand

Reply:
Oh, now I see. But that only made it worse.

Because this note is not related to anything specific, and you would think that all the methods in this article would work in Bedrock with this replacement. But the very first example, right after this note, will not work in Bedrock, because there is no `execute store`. And the method with the predicate (1.20.5) will not work either, and especially the advancement method. And only the method with tags and the second method with scoreboard will work in Bedrock.

So I still think it would be better to allocate a separate section for Bedrock. Since any of the Bedrock methods can be made simpler and more compact in Java, so there is no big reason to use these methods in Java.